tools/Ceres/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+
+project("Tools-Ceres")
+
+# Get Ceres (with GLOG)
+hunter_add_package(ceres-solver)
+find_package(Ceres CONFIG REQUIRED)
+include_directories("${CERES_INCLUDE_DIRS}")
+
+# Create GMM with different #define D and K values
+set(D_VALS 2 10) # NOTE unable to run 20, 32 due to memory error
+set(K_VALS 5 10 25 50) # removed 100, 200 as well
+# NOTE may be able to run more than is done here
+foreach (D_VAL ${D_VALS})
```

Comment:
Consider emitting these values to a file, to be picked up by run-all.
Also, is the crash at compile time or run time?   If the latter, let it happen -- it means ceres gets a black dot.
If at compile time, we may be able to help it out.

Reply:
Crash was compile time

Reply:
I didn't look into it at all (decided to leave and return to later) so it may be an easy fix - I'll have a look tomorrow.